README.md

```diff
@@ -83,6 +83,19 @@ earlier):
 
     sudo -u postgres psql bookbrainz -c "SELECT name FROM bookbrainz.editor"
 
+You are also required to set the password of PostgreSQL for your local.
```

Comment:
How about rephrasing it as such: "You are also required to set the password of your local PostgreSQL instance." ?